pkg/varnishcluster/controller/varnishcluster_grafana_dashboard.go

```diff
@@ -37,6 +37,10 @@ func (r *ReconcileVarnishCluster) reconcileGrafanaDashboard(ctx context.Context,
 		return nil
 	}
 
+	if instance.Spec.Monitoring.GrafanaDashboard.Title == "" {
+		instance.Spec.Monitoring.GrafanaDashboard.Title = fmt.Sprintf("%s varnish", instance.Name)
```

Comment:
nit: s/varnish/Varnish? since it's in the title

Reply:
So in this case `instance.Name` would be something like `vc-sun-location-services`. Do you think I should do some formatting on this to make it more readable? Maybe replacing the dashes with spaces or something?

Reply:
Hm, good question. I would prefer to not do formatting.
Maybe something like `Varnish (%s)`? so we'd have `Varnish (varnishcluster-name)`

Reply:
Sounds good, I like that.